### PR TITLE
Cassandra configuration

### PR DESCRIPTION
**Pull request by kimbyongjin**
This commit will add configuration for Cassandra and README documentation covering common commands and DB shell navigation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,13 +84,54 @@ Updates the information of a particular table entry located at provided id.
...
 Expects status code 200
 
-## Fake Data Generation
+#### Fake Data Generation
 
-``` WARNING! THE FOLLOWING COMMAND GENERATES ALL 100 MILLION+ RECORDS```
-``` THIS COMMAND IS INTENTIONALLY BROKEN BY DEFAULT ```
+ - ``` WARNING! THE FOLLOWING COMMAND GENERATES ALL 100 MILLION+ RECORDS```
+ - ``` THIS COMMAND IS INTENTIONALLY BROKEN BY DEFAULT ```
 To re-enable data generation, uncomment noted running script in file ``` data_gen/dataGen.js ```
```
Comment (rrdunn11) [inline README.md:91]: Good notes for future users or for your future self!

File: README.md [hunk 2]
```diff
@@ -84,13 +84,54 @@ Updates the information of a particular table entry located at provided id.
...
+
+### Initialize Cassandra bash shell
+
+ - From Hard Drive directory below user root directory:
+   - Will run Cassandra server.
+
+ ```cd usr/local/Cellar/cassandra/3.11.4/bin/```
+
+ - Then, from any terminal bash, run: ```cqlsh --color``` to open the Cassandra bash shell using colored output text.
+ - May omit the ```--color``` and just execute ```cqlsh``` as default command.
```
Comment (rrdunn11) [inline README.md:129]: I learned something new about the colored text option.

File: cassandra/schema.cql [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+CREATE TABLE photos(
+  photo_id INT PRIMARY KEY,
+  photo_url VARCHAR,
+  description VARCHAR,
+  is_verified BOOLEAN,
+  listing_id INT,
+);
+
+COPY photo_module.photos (photo_id,photo_url,description,is_verified,listing_id) FROM '/Users/nautilus/hackReactor/hrsf117-sdc/photodisplay-module/data_gen/fakeDataPhotosListings.csv' WITH HEADER= true AND REPORTFREQUENCY= '5';
```
Comment (rrdunn11) [inline cassandra/schema.cql:15]: What does the report frequency do here?

**Review by rrdunn11 — APPROVED** [review]
Looks good, Daniel! Very clear readme file.

Roger

File: README.md (hunk 2 above)
Comment (kimbyongjin) [reply README.md:129]: Yeah, it's really helpful for staying organized.

File: cassandra/schema.cql (hunk 3 above)
Comment (kimbyongjin) [reply cassandra/schema.cql:15]: Report frequency is set at .25 seconds by default. It provides feedback to the terminal while seeding a Cassandra DB. Shows how many total rows have been seeded, current rate, average rate, etc. I just wanted to lengthen the reporting interval to every 5 seconds. It still provides useful feedback. But, I don't need it so frequently. Even 5 seconds might be more frequent than necessary.

**Pull request merged**